testsuite/features/minxen_guests.feature

```diff
@@ -152,6 +160,13 @@ Feature: Be able to manage XEN virtual machines via the GUI
     And "test-vm2" virtual machine on "xen-server" should have 1 NIC using "default" network
     And "test-vm2" virtual machine on "xen-server" should have a "test-vm2_system.qcow2" xen disk
 
+@virthost_xen
+  Scenario: Show the Spice display
```

Comment:
"display" => "console", and:

We're trying to have unique scenario names, so we can use the `cucumber` command with a scenario name and run it in isolation.

Would it be possible to specify better in the scenario title in which extent it's different from previous `Show the Spice console` scenario?

Reply:
@Bischoff From the code perspective those two are identical, they just run on a different hypervisor. On some aspects that could make a difference, though on the graphical console side of things it won't.

Reply:
@cbosdo I don't care about the code perspective, I care about the testing perspective :smile: .

From a testing POV, if they run on two different hypervisors, then the scenario name should mention the hypervisor's name, like this:
```
Scenario: Show the Spice console for Xen
Scenario: Show the Spice console for libvirt
```

When it's done, we will be able to run one of them with e.g.:
```
$ cucumber -n "Show the Spice console for Xen"
```

Reply:
done

Reply:
I also added a commit to deduplicate the Xen and KVM scenario names